src/main/java/uk/gov/register/thymeleaf/ThymeleafView.java

```diff
@@ -21,17 +17,21 @@
 public class ThymeleafView extends View {
     protected final RequestContext requestContext;
     private final RegisterResolver registerResolver;
-    private final RegisterReadOnly register;
-    private Optional<String> registerTrackingId;
+    protected final RegisterReadOnly register;
```

Comment:
This isn't being used by any subclass of `ThymeleafView` - does it need to be `protected`?